setup.py

```diff
@@ -37,11 +37,7 @@ def get_version():
         'thoth.{subpackage}'.format(subpackage=p)
         for p in find_packages('thoth/')
     ],
-    zip_safe=False,
+    include_package_data=True,
     install_requires=get_install_requires(),
-    command_options={
-        'build_sphinx': {
-            'version': ('setup.py', VERSION),
-        }
-    }
```

Comment:
This looks like a wrong rebase?

Reply:
Oh my, I am so sorry, this is a terrible mistake, I must have deleted it while resolving a merge conflict. Thank you for spotting that.

Reply:
No problem at all. Thanks for the fix!